src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+body {
+  margin: 0;
+  font-family: $bodyFont;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 19px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 160px;
+    margin: 0 auto;
+    object-fit: contain;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $textColorDark;
+    margin-block: 27px 4px;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $textColorLight;
+    margin: 0;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $textColorDark;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $textColorLight;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: $textColorDark;
```

Comment:
In CSS/SCSS, name variables using kebab-case, like `text-color-dark`. Let's keep camelCase for JavaScript.